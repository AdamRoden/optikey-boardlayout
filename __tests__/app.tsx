import * as React from "react"
import App from "../src/app"
import renderSnapshot from "../src/testUtil/renderSnapshot"
test("app", () => {
  renderSnapshot(<App />, "inital app render")
})
