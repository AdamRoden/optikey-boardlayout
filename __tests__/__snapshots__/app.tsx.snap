// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app: inital app render 1`] = `
<div>
  hi App
  <div>
    hi panep
    <input
      onChange={[Function]}
      type="text"
      value="MMB2s Keyboard"
    />
  </div>
</div>
`;
